### PR TITLE
Fix all names

### PR DESCRIPTION
**Pull request by mattn**
## WHAT

```
s/dty1er/hidetatz/g
```

## WHY

I can't `go install`

### CODE REVIEW

File: printer/format_test.go [hunk 1]
```diff
@@ -3,8 +3,8 @@ package printer
 import (
 	"testing"
 
-	"github.com/dty1er/kubecolor/color"
-	"github.com/google/go-cmp/cmp"
+	"github.com/hidetatz/go-cmp/cmp"
```
Comment (hidetatz) [inline printer/format_test.go:6]:
```suggestion
	"github.com/google/go-cmp/cmp"
```

**Review by hidetatz — COMMENTED** [review]
Thank you

**Comment by mattn** [comment]
Done. Thanks your pointing my overdoing fixes.

**Review by hidetatz — APPROVED** [review]
LGTM
Thank you for your effort!

**Pull request merged**